README.md

```diff
@@ -151,8 +151,8 @@ Use [EIP-1102](https://eips.ethereum.org/EIPS/eip-1102) to obtain authorization
 The following code runs in response to a user-initiated action such as clicking a button to ensure the pop up is not blocked by the browser.
 
 ```typescript
-// Use eth_RequestAccounts
-ethereum.request('eth_requestAccounts').then((accounts: string[]) => {
+// Use eth_requestAccounts
+ethereum.send('eth_requestAccounts').then((accounts: string[]) => {
```

Comment:
@pedro-rosario why is this reverted? request is preferred to send

Reply:
That was a mistake, only line 154 should be changed, it is a typo fix

Reply:
That was a mistake, only line 154 should be changed, it is a typo fix